backend/tests/v5/unit/processors/teamspaces/projects/models/drawings/index.test.js

```diff
@@ -605,10 +606,13 @@ const testCreateDrawingThumbnail = () => {
 	describe('Create drawing thumbnail', () => {
 		test('Should store the thumbnail if it has been successfully generated', async () => {
 			const imageRef = generateUUID();
-			const imageBuffer = generateRandomString();
+			const imageBuffer = generateRandomBuffer();
+			const readStream = Readable.from(imageBuffer);
+			const mimeType = 'application/pdf';
```

Comment:
get constant from the aforementioned constant file here (applies for the other test cases too!)